### DISCUSSION TITLE
Add an easier way to report bugs

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5689] [Feature Requests]
When a bug occurs, making it as easy as possible to report it would be great.

Possibilities:
- Send us the bug report automatically - https://github.com/specify/specify7/issues/2599
- Add a button that opens a github issue, with a crash report attached
- Or at least add a button that copies the collection/user/url information so that internal testers could open github issues quicker

**Comment by grantfitzsimmons** [comment]
I'd argue this has been achieved. Should we close this @maxpatiiuk?

**Comment by maxpatiiuk** [comment]
My intention was that you can just press a keyboard shortcut, and it will copy the crucial system information (browser, collection, database, URL) so that you can easily paste then into a github issue/comment.

A more advanced version is where you press a keyboard shortcut, and it opens a new github issue page, but with the crash report already attached (or just system information if you want to report a bug that did not produce an error message)

The above two would be quite useful for internal testers. For external, exceptions be reported automatically (like in many modern applications) (https://github.com/specify/specify7/issues/2599) would help us detect what kind of errors users are encountering most often (i.e, time outs, misconfiguration, permission errors, real bugs)

**Comment by grantfitzsimmons** [comment]
Understood. Automatic error collection makes a lot of sense.

**Comment by grantfitzsimmons** [comment]
<img width="628" alt="Screenshot 2024-01-25 at 11 32 07 AM" src="https://github.com/specify/specify7/assets/37256050/71f45924-edfc-4bce-9364-537107839587">

Just something like this would be perfect.
